tools/depcheck/pkg/cmd/trace.go

```diff
@@ -185,8 +185,31 @@ func (o *TraceOptions) Run() error {
 		}
 	}
 
+	// possibleRoots are nodeNames that should be excluded
+	// from pruning or from analysis. We cannot rely on our
+	// given entrypoints as our source of roots because
+	// nodes created for package entrypoints are likely
+	// to have incomplete edge data based on the .go files
+	// immediately present in an entrypoint directory, and
+	// the direct import paths that they define.
+	possibleRoots := []string{}
```

Comment:
@deads2k `pkg/oc` was being affected by this. 
The only parent `pkg/oc` has is `./cmd/oc`. 
`./cmd` was entrypoint, but was not tied to `cmd/oc` because there are no .go files under that directory that directly bring in `cmd/oc`. `cmd/oc` then had no "to" edges, and ended up being pruned - bringing `pkg/oc` with it.